.git-blame-ignore-revs

```diff
@@ -1,2 +1,17 @@
+# Introduce Scalafmt
+774fb51d1365e1adef7fa71f09b1410941264f60
+
+# Migrate to new kind-projector symbol
+e053e287f092126328ffe7eb268281c81c041eed
+
+# Scalafmt updates
+54a6f26b178f490d3816c5eac6d2256228bb065e
+8d98a602a7c2b01b26e9935bfecb0e7f2bbd6737
+4836a55c872a4f931b7f8678f98d9009ed3df84a
+48e1a5a5af3bda6b2f99cd55bda00e14ef71aea7
+f025b0198d55d7f5c09ee976eec9e274d8dd0309
+1531bf8f3c1b1ecd7e37f1548c5e2756a9d52767
+6044f9569c62944aa2c2f4d6eca84cb9d7ca0d15
```

Comment:
Absolutely a minuscule: the next ignored commit is also related to scalafmt updates. But looks so lonely alone. Maybe let it unite with its friends in this block?

Reply:
That one was added automatically, so I didn't touch it. Even if we join it to this block, Steward may add more lonely ones in the future 😂 

https://github.com/typelevel/cats/commit/54195c43392f69b3cf638cdc5a211dd81db554ee